### PR TITLE
Option Menu - Logout and Your profile

### PR DESCRIPTION
**Pull request by ruvmello**
This is a option menu when you click the avatar icon. This PR has no tests as this is only visual change, the test coverage is the same because of this.

### CODE REVIEW

**Comment by jitsedesmet** [comment]
Maybe also use this PR to add some docs :slightly_smiling_face:

File: frontend/locales/en/common.ts [hunk 1]
```diff
@@ -51,6 +51,8 @@ const common = {
     edit: "edit",
 
     "no edition": "no edition",
+
+    logout: "logout",
```
Comment (jitsedesmet) [inline frontend/locales/en/common.ts:55]: The common file got some structure. I think this should be added in the user-management part.

**Review by jitsedesmet — COMMENTED** [review]
A little thing and then this should be good to go.

**Comment by ruvmello** [comment]
Will do the docs in another PR.

**Review by jitsedesmet — APPROVED** [review]
Good job. Should be ready. Don't forget the docs though ;)

**Pull request merged**